bot.user.js

```diff
@@ -595,10 +595,14 @@ var bot = (function() {
                     window.goalCoordinates = coordinatesOfClosestFood;
                     // Sprint
                     //use speed to go to larger clusters
-                    setAcceleration((window.currentFoodScore >= accelerationClusterSize) ? 1 : 0);
+					setAcceleration((window.currentFood.clusterScore >= accelerationClusterSize) 
+					?
+					(canvas.getDistance(window.getX(),window.getY(),window.currentFoodX,window.currentFoodY) <= Math.pow(window.getSnakeLength(), 2) / 2 && window.currentFood.distance > 375) 
+					? 
+					1 : 0 : 0);
```

Comment:
I'm having a hard time unwrapping these nested ternary operators... can you help?

I think it works out to something like this:
    if (window.currentFood.clusterScore >= accelerationClusterSize) {
        var distance = canvas.getDistance(window.getX(), window.getY(),
                                          window.currentFoodX, window.currentFoodY);
        var arbitraryCutoff = Math.pow(window.getSnakeLength(), 2) / 2;
        if (distance <= arbitraryCutoff && window.currentFood.distance > 375) {
            setAcceleration(1);
        }
        else {
            setAcceleration(0);
        }
    }
    else {
        setAcceleration(0);
    }